packages/terra-grid/CHANGELOG.md

```diff
@@ -0,0 +1,71 @@
+Terra Grid Changelog
+====================
+
+3.1.0 - 2017-06-20
```

Comment:
Total nit: I would vote towards formatting the date differently, maybe like React's changelog, which would use `(June 20, 2017)`